tueplots/fonts.py

```diff
@@ -18,3 +19,33 @@ def icml(*, usetex=False):
         "font.serif": ["Times"],
         "mathtext.fontset": "stix",  # free ptmx replacement
     }
+
+
+def beamer(*, usetex=False):  # for use with the MoML beamer template
```

Comment:
```suggestion
def beamer_moml():  
    """For use with the MoML beamer template."""
```

(`usetex` is not used here, and why not turn the comment into a proper docstring.)